Improve Document types.html setdigest Function (#20823)
prestodb/presto: presto-docs/src/main/sphinx/language/types.rst

    A KHyperLogLog is a data sketch that can be used to compactly represents the association of two
    columns. See :doc:`/functions/khyperloglog`.

SetDigest
---------

.. _setdigest_type:

``SetDigest``
^^^^^^^^^^^^^

A SetDigest (setdigest) is a data sketch structure used
in calculating `Jaccard similarity coefficient <https://wikipedia.org/wiki/Jaccard_index>`_
between two sets.

SetDigest encapsulates the following components:

- `HyperLogLog <https://wikipedia.org/wiki/HyperLogLog>`_
- `MinHash with a single hash function <http://wikipedia.org/wiki/MinHash#Variant_with_a_single_hash_function>`_

HyperLogLog: This is a probabilistic data structure used for the approximation of the number of distinct elements in

> a large dataset. It is based on the principle of observing the 'rarity' of certain patterns in a random process.

Reviewer: The word 'rarity' is enclosed in single vertical quotes. This is not RST formatting. Is this content copied from somewhere else? If so, it should be attributed. 

Why is the word marked like this? 
What do you mean by this? 
Is this marking needed, or can it be removed or changed?

HyperLogLog provides a way to estimate the number of distinct elements (the cardinality) with a high degree
of accuracy, using a relatively small amount of memory.

MinHash with a single hash function: This is a technique used to quickly estimate the similarity between two sets.

> It involves creating a 'signature' or 'sketch' for each set, which is a much smaller subset that still maintains the

Reviewer: The words 'signature' and 'sketch' are enclosed in single vertical quotes. This is not RST formatting. Is this content copied from somewhere else? If so, it should be attributed.

Why are the words marked like this?
What do you mean by this?
Is this marking needed, or can it be removed or changed?

same properties. The similarity between two sets can then be estimated by comparing their signatures.

> The HyperLogLog structure is used for the approximation of the distinct elements in the original set.

Reviewer: ```suggestion
The HyperLogLog structure is used to approximate the distinct elements in the original set.
```
Active voice is easier to read. Does this change alter the meaning of the sentence in a way you feel is less accurate?

> These components work together to provide an efficient and effective way to handle and compare large sets of data.

Reviewer: Does this sentence add to the value of the text? Suggest deleting it, unless it is important in a way I do not understand.

> SetDigests have the property of being additive, which means that you can merge multiple SetDigests together

Reviewer: ```suggestion
SetDigests are additive, so you can merge multiple SetDigests together
```
Does this change alter the meaning of the sentence in a way you feel is less accurate?
